handel/src/services/route53zone/index.ts

```diff
@@ -75,21 +75,11 @@ function getCompiledRoute53Template(ownServiceContext: ServiceContext<Route53Zon
  */
 
 export function check(serviceContext: ServiceContext<Route53ZoneServiceConfig>, dependenciesServiceContexts: Array<ServiceContext<ServiceConfig>>): string[] {
-    const errors = [];
+    const errors: string[] = checkPhase.checkJsonSchema(`${__dirname}/params-schema.json`, serviceContext);
 
     const params = serviceContext.params;
 
-    if (!params.name) {
-        errors.push(`${SERVICE_NAME} - 'name' parameter must be specified`);
-    } else if (!route53.isValidHostname(params.name)) {
```

Comment:
Sorry I meant just the first "if" part needed to go. The check inside the "else if" needs to stay. Sorry for not being clearer on that!